### ISSUE TITLE
ViewerJs button like (FullScreen & Presentation) are not working

### ISSUE THREAD

**Issue by AliMostafaHassan** [kogmbh/ViewerJS#197]
Dear All , i hope all of you are fine
when i put an iframe to show pdf using ViewerJS library in jquery popup , FullScreen & Presentation buttons are not wirking at all.

**Comment by jonnybradley** [comment]
Hi all,
We have a similar issue with ViewerJS in an iframe but not in a popup (although we're using bootstrap) - the full screen button doesn't work and the presentation one does, but after that the "close" presentation X icon is also not clickable - maybe connected?

TIA

**Comment by xavidp** [comment]
Hi

In our case, we seemed to be missing the params  **allowfullscreen webkitallowfullscreen** inside the iframe tag. Once added, fullscreen and presentation started to work as expected again.
 
Maybe it's the same case for you, AliMostafaHAssan?

**Comment by q2dg** [comment]
Yes, it is the same case. However, fullscreen and presentation behaves in a similar way...
